uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/PublicHoliday.tpl

```diff
@@ -0,0 +1,49 @@
+{if $action eq 1 or $action eq 2 or $action eq 8}
+    {include file="CRM/HRLeaveAndAbsences/Form/PublicHoliday.tpl"}
```

Comment:
This template is not part of this commit. This is probably a problem with the commit split.